Managers/UInputMan.cpp

```diff
@@ -357,11 +356,86 @@ void UInputMan::InputScheme::SetPreset(int schemePreset)
 		m_aInputMapping[INPUT_AIM_RIGHT].SetDirection(3, 0, JOYDIR_TWO);
 		m_aInputMapping[INPUT_AIM_RIGHT].SetPresetDesc("D-Pad Right");
 
+    case PRESET_P1DEFAULT:
+        m_aInputMapping[INPUT_L_UP].SetKey(KEY_W);
+        m_aInputMapping[INPUT_L_DOWN].SetKey(KEY_S);
+        m_aInputMapping[INPUT_L_LEFT].SetKey(KEY_A);
+        m_aInputMapping[INPUT_L_RIGHT].SetKey(KEY_D);
+        m_aInputMapping[INPUT_R_UP].SetKey(KEY_W);
+        m_aInputMapping[INPUT_R_DOWN].SetKey(KEY_S);
+        m_aInputMapping[INPUT_R_LEFT].SetKey(KEY_A);
+        m_aInputMapping[INPUT_R_RIGHT].SetKey(KEY_D);
+        m_aInputMapping[INPUT_FIRE].SetMouseButton(0);
+        m_aInputMapping[INPUT_PIEMENU].SetMouseButton(1);
+        m_aInputMapping[INPUT_JUMP].SetKey(KEY_W);
+        m_aInputMapping[INPUT_CROUCH].SetKey(KEY_S);
+        m_aInputMapping[INPUT_NEXT].SetKey(KEY_E);
+        m_aInputMapping[INPUT_PREV].SetKey(KEY_Q);
+        m_aInputMapping[INPUT_WEAPON_RELOAD].SetKey(KEY_R);
+        m_aInputMapping[INPUT_WEAPON_DROP].SetKey(KEY_G);
+        m_aInputMapping[INPUT_WEAPON_PICKUP].SetKey(KEY_F);
+        m_aInputMapping[INPUT_WEAPON_CHANGE_NEXT].SetKey(KEY_C);
+
```

Comment:
Missing `break;` at the end of each case.

Reply:
haha wow my bad.